rand/rand.go

```diff
@@ -0,0 +1,19 @@
+// Package rand is a convenience wrapper aroung golang rand
+package rand
+
+import (
+	"math/rand"
+	"time"
+)
+
+var r *rand.Rand
+
+// GetRandGenerator seeds a random number generator based on current time
+func GetRandGenerator() *rand.Rand {
+	if r != nil {
+		return r
+	}
+	source := rand.NewSource(time.Now().UnixNano())
```

Comment:
Could be nice to add a small commen like:
```
// The default number generator is deterministic
// so we need to use an unique seed to avoid generating the same numbers
// See https://gobyexample.com/random-numbers
```

Reply:
3309d66a